Makefile

```diff
@@ -85,22 +87,141 @@ test-daml:
 
 .PHONY: test
 test: test-daml test-ui
+	./scripts/verify-versions.sh
 
 ### *-=- Release -=-*
 
+.PHONY: tag
+tag:
+	@./scripts/tag-versions.sh \
+		$(VERSION) \
+		dabl-meta.yaml \
+		daml.yaml \
+		triggers/daml.yaml \
+		integrationTesting/daml.yaml\
+		exberry_adapter/setup.py \
+		$(UI_DIR)/package.json \
+		docs/local_development.md \
+		docs/damlhub_deployment.md
+	@echo "Tagged files... check results before committing"
+
 .PHONY: release
-release: test build
-	ddit release
+release: test package
+	ddit release --dry-run
```

Comment:
`--dry-run` will prevent this from accidentally deploying for real, until we're ready to turn it on